apps/web/src/screens/Dashboard/index.tsx

```diff
@@ -1,10 +1,15 @@
 import { FC } from 'react';
-import { Stack, Text } from '@walless/gui';
+import { Outlet } from 'react-router-dom';
+import { Stack } from '@walless/gui';
+
+import DashboardLayout from './DashboardLayout';
 
 export const DashboardScreen: FC = () => {
 	return (
 		<Stack>
-			<Text>DashboardScreen</Text>
+			<DashboardLayout>
+				<Outlet />
+			</DashboardLayout>
 		</Stack>
```

Comment:
is it possible to remove outside unused `Stack` and bring out `DashboardLayout`?

Reply:
Nice point, I forgot to remove it lol xD